.github/workflows/test.yml

```diff
@@ -27,7 +27,9 @@ jobs:
     - name: Unit test with pytest
       run: |
         pip install pytest pytest-cov
-        pytest tests/unit_tests --doctest-modules --junitxml=junit/unit-test-results.xml --cov=com --cov-report=xml --cov-report=html
+        pytest tests/unit_tests --doctest-modules --junitxml=junit/unit-test-results.xml
+        coverage run -m pytest tests/unit_tests
```

Comment:
Are we running `pytest` twice?
Can this match the implementation in the Civil Case API? 
[https://github.com/ministryofjustice/laa-civil-case-api/blob/main/.github/workflows/test.yml#L25](https://github.com/ministryofjustice/laa-civil-case-api/blob/main/.github/workflows/test.yml#L25)